src/workers/parser/getSymbols.js

```diff
@@ -199,7 +200,12 @@ function extractSymbol(path: SimplePath, symbols) {
       return;
     }
 
-    if (t.isProperty(path.parent)) {
+    if (
+      path.parent &&
+      t.isProperty(path.parent) &&
+      !t.isAssignmentPattern(path.getAncestor(3)) &&
+      !(t.isCallExpression(path.getAncestor(3)) && keyIsValue(path.parent))
```

Comment:
What's the use case for inAssignment?

Reply:
So the use case for the assignment pattern would be any time you have { blah } = {}, I may have to look at this again to make sure that it is working how I originally envisioned it.